src/proof/alethe/alethe_node_converter.cpp

```diff
@@ -218,6 +220,208 @@ Node AletheNodeConverter::postConvert(Node n)
       Trace("alethe-conv") << ".. converted to choice: " << converted << "\n";
       return converted;
     }
+    /* from uf */
+    case Kind::CARDINALITY_CONSTRAINT_OP:
```

Comment:
I think you should rephrase this so that this is a default case.

If one adds a new kind to cvc5, they shouldn't be obligated to add it here.

The CPC proof setup is organized in this way where only *supported* cases are listing in the code, not negative ones, see e.g. https://github.com/cvc5/cvc5/blob/main/src/proof/alf/alf_printer.cpp#L217 or https://github.com/cvc5/cvc5/blob/main/src/proof/alf/alf_node_converter.cpp#L690.

This makes it easier to maintain going forward.

Reply:
Good point. I'll update.